### PR TITLE
Unique search identifier for each request

### PR DESCRIPTION
**Pull request by xoriole**
Unique search identifier for each request helps to identify the response.

### CODE REVIEW

File: Tribler/Core/Modules/restapi/search_endpoint.py [hunk 1]
```diff
@@ -96,11 +100,16 @@ def render_GET(self, request):
             request.setResponseCode(http.BAD_REQUEST)
             return json.dumps({"error": "Trying to query for unknown type of metadata"})
 
+        search_uuid = self.get_uuid(request.args)
+        if not search_uuid:
```
Comment (devos50) [inline Tribler/Core/Modules/restapi/search_endpoint.py]: Can we make this optional? There are times when I'm using the REST API with curl and it is inconvenient if I have to specify a UUID every time.

**Comment by xoriole** [comment]
retest this please

**Comment by xoriole** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
(just testing the new libtorrent binary on windows)

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Pull request merged**